php-checks/src/test/resources/checks/CodeFollowingJumpStatementCheck.php

```diff
@@ -181,3 +182,7 @@ function return_in_try() {
     dosmth();
   }
 }
+
+
+throw new Exception();
+?>
```

Comment:
Before this new test case, I would also add a test case like:
```
function containingInlineHtml() {
  throw new Exception();
  ?>unreachable<?php // Noncompliant
}
```

Reply:
This is already coverd by
```
function tagAfterReturn() {
  return;
  ?><?php // Noncompliant
}
```

Reply:
Indeed!